tmail-backend/tmail-third-party/openpaas/src/main/java/com/linagora/tmail/contact/OpenPaasContactsConsumer.java

```diff
@@ -120,9 +119,15 @@ private Mono<EmailAddressContact> messageConsume(AcknowledgableDelivery ackDeliv
         return Mono.just(messagePayload)
             .map(this::parseContactAddedRabbitMqMessage)
             .flatMap(this::handleMessage)
-            .doOnSuccess(input -> ackDelivery.ack())
-            .doOnError(e -> ackDelivery.nack(REQUEUE_ON_NACK))
-            .onErrorResume(e -> Mono.error(new RuntimeException("Failed to consume OpenPaaS added contact message", e)));
+            .doOnSuccess(result -> {
+                LOGGER.warn("Consumed contact successfully '{}'", result);
+                ackDelivery.ack();
+            })
+            .onErrorResume(error -> {
+                LOGGER.error("Error when consume message '{}'", messagePayload, error);
```

Comment:
Please let's not log the full payload...

Reply:
How about on success?